pkg/tsdb/azuremonitor/types/types.go

```diff
@@ -28,14 +28,16 @@ type AzRoute struct {
 }
 
 type AzureMonitorSettings struct {
-	SubscriptionId               string `json:"subscriptionId"`
-	LogAnalyticsDefaultWorkspace string `json:"logAnalyticsDefaultWorkspace"`
-	AppInsightsAppId             string `json:"appInsightsAppId"`
+	SubscriptionId               string                              `json:"subscriptionId"`
+	LogAnalyticsDefaultWorkspace string                              `json:"logAnalyticsDefaultWorkspace"`
+	AppInsightsAppId             string                              `json:"appInsightsAppId"`
+	CustomizedCloudSettings      AzureMonitorCustomizedCloudSettings `json:"customizedCloudSettings"`
```

Comment:
I would make this optional:

```suggestion
	CustomizedCloudSettings      *AzureMonitorCustomizedCloudSettings `json:"customizedCloudSettings,omitempty"`
```